man/passwd.1.xml

```diff
@@ -94,23 +94,16 @@
       </para>
 
       <para>
-	Then, the password is tested for complexity. As a general guideline,
-	passwords should consist of 6 to 8 characters including one or more
-	characters from each of the following sets:
+	Then, the password is tested for length.  As a general guideline,
+	passwords should be long.  It's fine to use simple character sets,
+	such as passwords consisting only of lowercase letters.  For a
+	password consisting only of lowercase English letters randomly
+	chosen, and a length of 64, there are 27^64 (approximately 2^304)
```

Comment:
you say "only lowercase English letters", but then say 27.  presumably you're including space in the count.

Reply:
Oops, miscount here :)  For some reason, `27 - 1 = 27` yesterday.